packages/s/simpleini/xmake.lua

```diff
@@ -0,0 +1,24 @@
+package("simpleini")
+    set_kind("library", {headeronly = true})
+    set_homepage("https://github.com/brofield/simpleini")
+    set_description("Cross-platform C++ library providing a simple API to read and write INI-style configuration files.")
+    set_license("MIT")
+
+    set_urls("https://github.com/brofield/simpleini/archive/refs/tags/$(version).tar.gz",
+             "https://github.com/brofield/simpleini.git")
+    add_versions("v4.19", "dc10df3fa363be2c57627d52cbb1b5ddd0689d474bf13908e822c1522df8377e")
+
+    add_deps("convertutf", {optional = true})
```

Comment:
please do not use optional, you should add `add_configs()` to enable/disable it and define macro.

https://github.com/brofield/simpleini/blob/142beb4ecef7b50887ae99e7dfcbc09f779372cd/SimpleIni.h#L3008